upgrade chart for v2 (#81)
localstack/helm-charts: charts/localstack/README.md

|------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------|
| `debug`                                              | Specify if debug logs should be enabled                                                                                                                                                                                               | `false`                                                 |
| `kinesisErrorProbability`                            | Specify to randomly inject ProvisionedThroughputExceededException errors into Kinesis API responses                                                                                                                                   | `nil` (Localstack Default)                              |

> | `startServices`                                      | Comma-separated list of AWS CLI service names which should be loaded right when starting LocalStack. Per default, all services are loaded and started on the first request for that service.                                          | `nil` (Localstack Default)                              |

Reviewer: ```suggestion
| `startServices`                                      | Comma-separated list of AWS CLI service names which should be loaded right when starting LocalStack. If not set, each service is loaded and started on the first request for that service.                                          | `nil` (Localstack Default)                              |
```

I could see the wording here being a bit confusing.
